include/bout/run_metrics.hxx

```diff
@@ -0,0 +1,65 @@
+/**************************************************************************
+ * Copyright 2010 B.D.Dudson, S.Farley, M.V.Umansky, X.Q.Xu
+ *
+ * Run Metrics class by J. T. Parker 2019
+ *
+ * Contact: Ben Dudson, bd512@york.ac.uk
+ *
+ * This file is part of BOUT++.
+ *
+ * BOUT++ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * BOUT++ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with BOUT++.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **************************************************************************/
+
+/// Run Metrics class
+///
+/// The `Run Metrics` class holds non-physics information about the job execution,
+/// such as wall clock times and number of iterations.
+
+class RunMetrics;
+
+#ifndef __RUNMETRICS_H__
+#define __RUNMETRICS_H__
+
+#include "datafile.hxx"
+
+
+class RunMetrics {
+  public:
+  BoutReal simtime;      // cumulative simulation time
```

Comment:
Please could you move the comments to the line before the variable and use triple-slashes for doxygen: `///`

Reply:
Thanks - I've Doxyfied them.